custom/export_demographics.php

```diff
@@ -1,23 +0,0 @@
-<html>
```

Comment:
I suspect it is no longer used but cannot be sure.  However it may still be useful as a hook and example use of it.